dags/openshift_nightlies/config/install/hypershift/none-type.json

```diff
@@ -9,7 +9,8 @@
     "openshift_network_type": "OpenShiftSDN",
     "openshift_worker_instance_type": "m5.4xlarge",
     "number_of_hosted_cluster": 10,
-    "control_plane_operator_image": "quay.io/hypershift/hypershift:latest",
+    "control_plane_operator_image": "",
```

Comment:
its the default set already [here](https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/tasks/install/hypershift/defaults.json#L36), so lets remove this from install input json files.